typings/index.d.ts

```diff
@@ -1777,15 +1777,17 @@ declare module 'discord.js' {
 	type GuildChannelResolvable = Snowflake | GuildChannel;
 
 	type GuildCreateChannelOptions = {
-		type?: 'text' | 'voice' | 'category'
+		name?: string;
```

Comment:
This would allow something like `guild.channels.create('name', { name: 'another name' });` which is not valid and could confuse or mislead people.
You could either make a new type `GuildChannelCloneOptions` or something, or just `& { name?: string }` the type in the method declaration.

Reply:
Done in 0cdbf14 👍